scraper/src/strategies/default_strategy.py

```diff
@@ -121,6 +121,10 @@ def get_records_from_dom(self, current_page_url=None):
                     content is None or content == "") and current_level == 'content':
                 continue
 
+            # We do not want to keept level without current_level. Example h1 is an empty element.
```

Comment:
```suggestion
            # We do not want to keep levels without current_level because they don't provide semantic value. For example, a h1 that only contains a SVG and doesn't have any text content should be skipped.
```